### DISCUSSION TITLE
How can I  start Rubberduck?

### DISCUSSION THREAD

**Discussion by gittar** [rubberduck-vba/Rubberduck#6079] [Q&A] [answered]
I have installed the latest stable release of Rubberduck. How can I actually start it? Do I have to manually add an addIn to my excel file?

**Comment by Vogel612** [comment, marked as answer]
The installation should automatically register Rubberduck with the VBE. The usual keyboard shortcut to open the VBE is Alt+F11 when in a document. If everything went well, you should see the Rubberduck status bar in the top left of the newly opened VBE.

**Reply by gittar** [reply]
Worked perfectly. I had VBE open during installation of Rubberduck. After closing it and opening it again, Rubberduck appeared. Thanks!
